internal/store/postgres/postgres.go

```diff
@@ -13,13 +13,117 @@ import (
 )
 
 // New returns a new Service.
-func New(db *sql.DB) *store.Service {
+func New(db *sql.DB) (*store.Service, error) {
+	if _, err := db.Exec(eventTableCreationQuery); err != nil {
```

Comment:
Could we replace these with a for loop or would that be worse?

Reply:
fixed